providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/e2e/BaseStepDefinitions.java

```diff
@@ -1,29 +1,22 @@
 package dev.openfeature.contrib.providers.flagd.e2e;
 
-import dev.openfeature.contrib.providers.flagd.FlagdOptions;
-import dev.openfeature.contrib.providers.flagd.FlagdProvider;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+
 import dev.openfeature.sdk.Client;
 import dev.openfeature.sdk.EvaluationContext;
 import dev.openfeature.sdk.FlagEvaluationDetails;
 import dev.openfeature.sdk.ImmutableContext;
-import dev.openfeature.sdk.OpenFeatureAPI;
 import dev.openfeature.sdk.Reason;
 import dev.openfeature.sdk.Structure;
 import dev.openfeature.sdk.Value;
-import io.cucumber.java.BeforeAll;
-import io.cucumber.java.en.Given;
-import io.cucumber.java.en.Then;
-import io.cucumber.java.en.When;
 
-import java.util.HashMap;
-import java.util.Map;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+public class BaseStepDefinitions {
 
-public class StepDefinitions {
+    protected static Client client;
```

Comment:
suggestion

Shall we do this through a setting 

```java
    public void setClient(Client client) {
        this.client = client;
    }
```

Which then can be called from before all of the sub classes ?

Reply:
No longer applicable, I've removed the double test suites.